docs/contributing/dependencies/index.md

```diff
@@ -0,0 +1,116 @@
+---
+sidebar_custom_props:
+  access: bitwarden
+---
+
+# Dependency Management
+
+Bitwarden uses [Renovate](https://www.mend.io/renovate/) for automating dependency updates. Renovate
+will automatically create pull requests for dependencies on a weekly cadence. Security updates will
+generate pull requests immediately.
+
+## Ownership
+
+Bitwarden's repositories falls under two categories, team-owned and shared repositories.
+
+### Team-Owned repositories
+
+Team owned repositories are "owned" by a single team from a dependency standpoint. The assigned team
+is responsible for reviewing, approving and merging dependency updates. Some reason a repository
+might be team-owned is because it's primarily developed by that team, or to balance out the number
+of dependencies teams have to manage.
+
+Some examples are [`directory-connector`][dc] which is owned by the _Admin Console_ team and
+[`key-connector`][kc] which is owned by the _Auth_ team.
```

Comment:
💭 Might be tenuous to mention teams in documentation as they could change.

Reply:
True, but it's nice to give examples. We can revisit it should the need arise. We probably need a place to store the "ownership" doc which might be a better reference long term.